src/codegen/pythoncg.cpp

```diff
@@ -716,6 +719,10 @@ wxString PythonCodeGenerator::GetCode(PObjectBase obj, wxString name, bool silen
     PythonTemplateParser parser(obj, _template, m_i18n, m_useRelativePath, m_basePath, m_imagePathWrapperFunctionName);
     wxString code = parser.ParseTemplate();
 
+    // add spaces around bitwise or shift operator
+    if (code.Find(wxT(" | ")) == wxNOT_FOUND)
+        code.Replace(wxT("|"), wxT(" | "));
+
```

Comment:
This is very dangerous, this replacement is done on the fully evaluated code template without knowing and respecting its syntax. This DOES break user code, you can verify this easily by using e.g. a `wxStaticText` with a label `My|Label`, a space padding gets added around the pipe.